### PR TITLE
[AF-1634] Social framework removal

### PR DESCRIPTION
**Pull request by caponetto**
Pull requests associated with this change:
- kiegroup/appformer#701
- kiegroup/kie-wb-distributions#923
- kiegroup/kie-wb-common#2646
- kiegroup/drools-wb#1141
- kiegroup/optaplanner-wb#335
- kiegroup/jbpm-wb#1350

### CODE REVIEW

**Comment by caponetto** [comment]
Jenkins retest this

File: kie-wb-common-cli/kie-wb-common-cli-tools/kie-wb-common-cli-forms-migration/pom.xml [hunk 1]
```diff
@@ -90,6 +90,10 @@
       <groupId>org.uberfire</groupId>
       <artifactId>uberfire-nio2-model</artifactId>
     </dependency>
+    <dependency>
```
Comment (domhanak) [inline kie-wb-common-cli/kie-wb-common-cli-tools/kie-wb-common-cli-forms-migration/pom.xml]: Could you please explain why is this addition necessary?

Comment (caponetto) [reply kie-wb-common-cli/kie-wb-common-cli-tools/kie-wb-common-cli-forms-migration/pom.xml]: Hi @domhanak, thanks for reviewing this one!
I had to add this dependency after removing the social framework dependencies, otherwise, the compilation would fail.
However, this dependency is needed only in test scope, so I've moved it. Please check my new commit.

**Comment by domhanak** [comment]
jenkins retest this

**Comment by domhanak** [comment]
jenkins retest this

**Comment by ederign** [comment]
failures are not related.
